### PR TITLE
Added X-Robots-Tag to stop the webpage from crawling

### LINKED ISSUE #78: set xrobotstag in firebase.json
this webpage should not be crawled

Reference: 
https://developers.google.com/search/docs/crawling-indexing/robots-meta-tag#xrobotstag

### PR DESCRIPTION
**Pull request by aswin2108**
###Related Issue(s)
fixes #78 

## Describe Your Changes
Added the `X-Robots-Tag` to `noindex` in the firebase.json file, to stop the webpages from getting crawled. Referred to this [material](https://developers.google.com/search/docs/crawling-indexing/robots-meta-tag#xrobotstag). 

@kachun333 Can you pls check it?

### CODE REVIEW

File: firebase.json [hunk 1]
```diff
@@ -15,6 +15,15 @@
           }
         ]
       },
+      {
+        "source": "**/*.html",
+        "headers": [
+          {
+            "key": "X-Robots-Tag",
+            "value": "noindex"
```
Comment (kachun333) [inline firebase.json:23]:
```suggestion
            "value": "noindex, nofollow"
```

**Review by kachun333 — CHANGES_REQUESTED** [review]
please add no follow as well, I will test the changes after that. Thanks

File: firebase.json (hunk 1 above)
Comment (aswin2108) [reply firebase.json]: Added!

**Review by kachun333 — DISMISSED** [review]

**Review by kachun333 — APPROVED** [review]
I have tested and it works. Thanks.

**Pull request merged**